waffle_hub/dataset/adapter/autocare_dlt.py

```diff
@@ -87,8 +87,11 @@ def export_autocare_dlt(self, export_dir: Union[str, Path]) -> str:
     export_dir = Path(export_dir)
 
     train_ids, val_ids, test_ids, unlabeled_ids = self.get_split_ids()
+    background_ids = self.get_background_ids()
 
     if self.task in TaskType:
+        if self.task != TaskType.CLASSIFICATION:
+            train_ids += background_ids
```

Comment:
It can occur unexpected results. Because, we are not sure that `autocare_dlt` provides background image training. (same issue in tranformers)

Reply:
I'm agree with your opinion.I'm not sure about `autocare_dlt`, so I left the background related work as `TODO`.
However, it has been confirmed that it works normally in `transforemrs`. So, I didn't modify `transformers` source code